core/shared/utils/bh_vector.c

```diff
@@ -143,24 +143,26 @@ bh_vector_insert(Vector *vector, uint32 index, const void *elem_buf)
 {
     size_t i;
     uint8 *p;
+    bool ret = false;
 
     if (!vector || !elem_buf) {
         LOG_ERROR("Insert vector elem failed: vector or elem buf is NULL.\n");
-        return false;
+        goto RETURN;
     }
 
     if (index >= vector->num_elems) {
         LOG_ERROR("Insert vector elem failed: invalid elem index.\n");
-        return false;
+        goto RETURN;
```

Comment:
Can we rename the label to lowercase? Had better be consistent with current coding style. And label name of uppercase is not easy to read.